src/mlmd/LineageCardRow.tsx

```diff
@@ -166,14 +202,80 @@ export class LineageCardRow extends React.Component<LineageCardRowProps> {
   }
 
   private checkRadio(): JSX.Element {
-    if (!this.props.hideRadio) {
-      return <div><input type='radio' className='form-radio' name='' value='' onClick={this.handleClick} /></div>;
+    if (this.props.hideRadio) {
+      return <div className='noRadio' />;
     }
-    return <div className='noRadio' />;
+
+    return (
+      <div className={'form-radio-container'}>
+        <input
+          type='radio'
+          className='form-radio'
+          name=''
+          value=''
+          onClick={this.handleClick}
+          checked={this.props.isTarget}
+          readOnly={true}
+        />
+        <input
```

Comment:
Why are there 2 radios?

Reply:
One is used to provide a selection hint when the card is hovered.

The natural hover state of the material radio button does not show a hint so adding a `:hover` isn't very obvious what hovering is actually doing. Now that there's always a selected target card, there's a grey hint when you hover the card that goes away when you hover the link to indicate targeting vs following the hyperlink.

See the effect in the screencast.

Reply:
Looks fantastic! Thanks :)